validator/client/fake_validator_test.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/rpc/v1"
+)
+
+var _ = Validator(&fakeValidator{})
+
+type fakeValidator struct {
+	InitializeCalled bool
+
```

Comment:
could you remove all the empty lines ?

Reply:
Yeah, the empty lines were my way of grouping these methods, but it's totally unnecessary and harder to read this way.
Thanks!